lib/hergetto/structs/message.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Hergetto.Structs.Message do
+  @moduledoc """
+  This module represents a chat message.
+  """
+
+  defstruct [:message, :sender]
```

Comment:
I just remembered one important item in this list, datetime. Might be handy to have

Reply:
True